GuitarString.h

```diff
@@ -0,0 +1,47 @@
+/******************************************************************
+    GuitarString.h
+
+    structure to encapsulate a note on the guitar string
+******************************************************************/
+#pragma once
+#include "LeakedObjectDetector.h"
+#include "Defines.h"
+#include "Note.h"
+#include "GuitarString.h"
```

Comment:
You are including "GuitarString.h" **in** "GuitarString.h".
that doesn't make sense
- [ ] revise